### DISCUSSION TITLE
Elapsed time

### DISCUSSION THREAD

**Discussion by BrianKaye47** [rpm-software-management/dnf5#1133] [General]
It would be nice to have the date, time and elapsed time displayed for a command instance. I do it for dnf3 with a shell script function but was never able to handle the group sub-command.

**Comment by ppisar** [comment]
I thing "script" (logs every line written to a terminal with a time stamp) or "time" (reports how long a program run) are better tools that work with any command. I don't see a reason why DNF5 should implement this feature.

**Reply by BrianKaye47** [reply]
It would be a simple change. Record starting time and end time and subtract the two for elapsed time. If one keeps a terminal window open  for updates, you could easily see when the last updates were done.

**Reply by ppisar** [reply]
That data are already stored in /var/log/dnf5.log log file you can use for your reference.
